apps/src/applab/dropletConfig.js

```diff
@@ -25,7 +26,7 @@ var COLOR_YELLOW = '#FFF176';
  */
 function getScreenIds() {
   var ret = $("#designModeViz .screen").map(function () {
```

Comment:
we don't want to pull these ids from divApplab because we use this dropdown in design mode, for the screen selector in the DesignToggleRow.